tools/check/ut.go

```diff
@@ -795,9 +795,9 @@ func buildTestBinary(pkg string) error {
 	// go test -c
 	var cmd *exec.Cmd
 	if coverprofile != "" {
-		cmd = exec.Command("go", "test", "-c", "-cover", "-vet", "off", "-o", testFileName(pkg))
+		cmd = exec.Command("go", "test", "-ldflags=-w", "-gcflags=\"-N -l\"", "-c", "-cover", "-vet", "off", "-o", testFileName(pkg))
```

Comment:
This doesn't work because `buildTestBinary` is not really used by the CI script
You need to change `buildTestBinaryMulti`.